blogs/packet_drops/packets_drop.md

```diff
@@ -0,0 +1,155 @@
+# Network Observability Real-Time Per Flow Packets Drop
+
+![logo](./images/packets_drop_logo.png)
+
+By: Amogh RD, Julien Pinsonneau and Mohamed S. Mahmoud
+
+In OCP ensuring efficient packet delivery is crucial for maintaining smooth
+communication between applications. However, due to various factors such
+as network congestion, misconfigured systems, or hardware limitations,
+packets might occasionally get dropped. Detecting and diagnosing these
+packet drops is essential for optimizing network performance and
+maintaining a high quality of service.
+This is where eBPF (extended Berkeley Packet Filter) comes into play
+as a powerful tool for real-time network performance analysis.
+In this blog, we'll take a detailed look at how network observability
+using eBPF can help in detecting and understanding packet drops,
+enabling network administrators and engineers to proactively
+address network issues.
+
+## Detecting Packet Drops with eBPF
+
+eBPF enables developers to set up tracepoints at key points within the network
+stack. These tracepoints can help intercept packets at specific events,
+such as when they are received, forwarded, or transmitted.
+By analyzing the events around packet drops, you can gain insight into the
+reasons behind them.
+In network observability we are using `tracepoint/skb/kfree_skb` tracepoint hook
+to detect when packets are dropped, the reason for packets drop and reconstruct
+the flow and enrich it with drop metadata such as packets and bytes statistics,
+for TCP only the latest TCP connection state as well as the TCP connection flags
+are added.
+Packets drop ebpf hook supports TCP, UDP, SCTP, ICMPv4 and ICMPv6 protocols.
+There are two main categories for packet drops, core subsystem drops which cover
+most of the host drop reasons; for the complete list please refer to
+https://github.com/torvalds/linux/blob/master/include/net/dropreason-core.h
+and OVS based drops which is a recent kernel enhancement; for reference please 
+checkout the following link
+https://git.kernel.org/pub/scm/linux/kernel/git/netdev/net-next.git/tree/net/openvswitch/drop.h.
```

Comment:
```suggestion
There are two main categories for packet drops. The first category, core subsystem drops, covers
most of the host drop reasons; for the complete list please refer to
https://github.com/torvalds/linux/blob/master/include/net/dropreason-core.h. Second, there are OVS-based drops, which is a recent kernel enhancement; for reference please 
checkout the following link
https://git.kernel.org/pub/scm/linux/kernel/git/netdev/net-next.git/tree/net/openvswitch/drop.h.
```

Reply:
I suggest this punctuation instead. There is a comma splice and run-on sentence in the current construction.